client/web/src/insights/pages/dashboards/dashboard-page/components/dashboards-content/utils/find-dashboard-by-url-id.ts

```diff
@@ -0,0 +1,28 @@
+import { InsightDashboard, isVirtualDashboard } from '../../../../../../core/types'
+import { isSettingsBasedInsightsDashboard } from '../../../../../../core/types/dashboard/real-dashboard'
+
+/**
+ * Returns dashboard configurations by URL query param - dashboardID.
+ *
+ * @param dashboards - list of all reachable dashboards
+ * @param dashboardID - possible dashboard id from the URL query param.
+ */
+export function findDashboardByURLId(
```

Comment:
super nit:

```suggestion
export function findDashboardByUrlId(
```

or 

```suggestion
export function findDashboardByURLID(
```